Hannah-Sten/TeXiFy-IDEA
# Add \newcommand first parameter manually to structure view when braces are left out

Fixes #1130

We can not do this with the value of \newcommand (the {42} in the example below) as it may be a group, or an other command, or even just a single number.

_To test this PR:_

Check that \magik appears in the structure view:
```latex
\documentclass{article}
\newcommand\magik{42}
\begin{document}
    \magik
\end{document}
```

- [x] Add to wiki (also add that braces are recommended)
